storage/src/rotate_encryption_key.php

```diff
@@ -29,25 +29,20 @@
 /**
  * Change the encryption key used to store an existing object.
  *
- * @param string $bucketName The name of your Cloud Storage bucket.
- * @param string $objectName The name of your Cloud Storage object.
- * @param string $oldBase64EncryptionKey The Base64 encoded AES-256 encryption
+ * @param string $bucketName The name of your Cloud Storage bucket. (e.g. 'my-bucket')
+ * @param string $objectName The name of your Cloud Storage object. (e.g. 'my-object')
+ * @param string $oldBase64EncryptionKey The Base64 encoded AES-256 encryption (e.g. 'TIbv/fjexq+VmtXzAlc63J4z5kFmWJ6NdAPQulQBT7g=')
  *     key originally used to encrypt the object. See the documentation on
  *     Customer-Supplied Encryption keys for more info:
  *     https://cloud.google.com/storage/docs/encryption/using-customer-supplied-keys
- * @param string $newBase64EncryptionKey The new base64 encoded encryption key.
+ * @param string $newBase64EncryptionKey The new base64 encoded encryption key. (e.g. '0mMWhFvQOdS4AmxRpo8SJxXn5MjFhbz7DkKBUdUIef8=')
```

Comment:
This may also need to be broken into multiple lines like this:

```suggestion
 * @param string $newBase64EncryptionKey The new base64 encoded encryption key.
          (e.g. '0mMWhFvQOdS4AmxRpo8SJxXn5MjFhbz7DkKBUdUIef8=')
```